example/vanilla-k8s-block-driver/vsphere.conf

```diff
@@ -1,6 +1,7 @@
 [Global]
 cluster-id = "unique-kubernetes-cluster-id"
 volumemigration-cr-cleanup-intervalinmin = "120"
+csi-auth-check-intervalinmin = "30"
```

Comment:
Can we reduce it to 5 mins? I believe that the auth check workflow is not expensive.

Reply:
Default value is 5 minutes.  Here we just give an example to show how to set the interval other than the default value.

Reply:
If we set `csi-auth-check-intervalinmin = "30"` in example conf file, wouldn't this prompt the users to deploy csi driver with 30 minutes as interval. 
It would be nice to have to this set to 5 which will be as same as default value

Reply:
Agree with @chethanv28 . Many customers literally just copy past from our example configuration settings, so its important to make sure that the example file reflects the defaults in the code.

Reply:
Fixed,